packages/kbn-plugin-generator/template/package.json.ejs

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "build": "yarn plugin-helpers build",
```

Comment:
Can this be removed since we're directing people to use `yarn plugin-helpers build`?

Reply:
It can, just figured it served a little like additional documentation.